lib/src/components/NxCollapsibleItems/NxCollapsibleItems.tsx

```diff
@@ -66,7 +66,7 @@ const NxCollapsibleItems: NxCollapsibleItemsFC = function NxCollapsibleItems(pro
   return (
     /* eslint-disable-next-line jsx-a11y/role-supports-aria-props */
     <div className={treeViewClasses}
-         id={treeViewId}
+         id={id}
```

Comment:
This can be removed and `id` can be removed from the `props` destructuring, so that it ends up getting passed through the `otherProps`.

Reply:
Removed here 740f8614548ae0a6a9a03b5afe803d17385f2e73.